### ISSUE TITLE
Sending email failes if email contain special character

### ISSUE THREAD

**Issue by amandruby** [sj26/mailcatcher#463]
Hello,
I have created an application in which some users have email which contains special characters, and when we deliver mail to those users it fails with error message : -

```
*** Error receiving message: {:sender=>"<from@example.com>", :recipients=>["<som\xC3\xA9body@here.com>"], :source=>"Date: Sat, 12 Jun 2021 15:39:44 +0530\r\nFrom: <from@example.com>\r\nTo: =?UTF-8?B?c29tw6lib2R5?=@here.com\r\nMessage-ID: <*************>\r\nSubject: "xyz"}

    Exception: "\xC3" from ASCII-8BIT to UTF-8
```

**Comment by sj26** [comment]
MailCatcher does not support utf8 or 8bitmime, it follows the original smtp standard which is ascii (7 bit) only. Your smtp or mail libraries are not creating correctly formatted messages. What are you using?

**Comment by sj26** [comment]
This should be fixed by #467.

**Comment by sj26** [comment]
Your SMTP client and mail library should be correctly encoding mail message bodies to only contain 7-bit ASCII characters, per [RFC5321](https://datatracker.ietf.org/doc/html/rfc5321). Wider characters can be supplied in smtp by using quoted strings, and in mail bodies by using encodings like quoted printable, and most smtp clients and mail libraries will handle this transparently. I suggest checking that your SMTP client and mail library are conforming to standards. Many SMTP servers are quite lenient with these sorts of issues, so you might be fine. But you might also have trouble sending email over the wider Internet.
